spec/checkPullRequestReviewSpec.js

```diff
@@ -77,6 +82,58 @@ describe('Pull Request Review Module', () => {
       };
     });
 
+    describe('When reviewer requests changes and LGTM label ' +
+     'was already added to the pull request.', ()=>{
+      beforeEach(async () => {
+        const label = {
+          id: 248679580,
+          node_id: 'MDU6TGFiZWwyNDg2Nzk1ODA=',
+          url: 'https://api.github.com/repos/oppia/oppia/labels/PR:%20LGTM',
+          name: 'PR: LGTM',
+          color: '009800',
+        };
+        // Set the payload action and label which will simulate removing
+        // the LGTM label.
+        payloadData.payload.label = label;
+        payloadData.payload.pull_request.requested_reviewers = [
+          { login: 'reviewer1' },
+          { login: 'reviewer2' },
+        ];
+        payloadData.payload.pull_request.assignees = [];
+        // Set project owner to be pr author.
+        payloadData.payload.pull_request.user.login = 'kevintab95';
+        reviewPayloadData.payload.pull_request.labels.push(label);
```

Comment:
Add an `afterAll` block to return the payload to its initial state following the above modification.

Reply:
@jameesjohn Sir Changes have been done. Could you take a look at them and confirm if any change is required in them. Thanks.

Reply:
You should also revert the labels.